.vsts-ci.yml

```diff
@@ -12,17 +12,19 @@ variables:
     PB_PublishBlobFeedUrl: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
     _DotNetPublishToBlobFeed: true
 
-phases:
+jobs:
```

Comment:
The variables above need to be defined using the array syntax.  You can either define variables using the object syntax or the array syntax, but you can't mix n' match.

https://github.com/dotnet/arcade/blob/master/Documentation/AzureDevOps/PhaseToJobSchemaChange.md#phase-to-job-schema-transition

We've settled on the array syntax because the object syntax doesn't support variable groups.  In its current state, I think all of the above variables will just get silently ignored.

Reply:
Done